server/bundles/io.cloudbeaver.service.security/src/io/cloudbeaver/service/security/CBEmbeddedSecurityController.java

```diff
@@ -1910,12 +1912,13 @@ private boolean isTokenExpired(Timestamp tokenExpiredDate) {
     @Override
     public SMAuthInfo finishAuthentication(@NotNull String authId) throws DBException {
         SMAuthInfo authInfo = getAuthStatus(authId);
-        return finishAuthentication(authInfo, false);
+        return finishAuthentication(authInfo, false, false);
```

Comment:
`return finishAuthentication(authInfo, false, false);`
It looks like this feature will never work with SSO